datasets/bdi_benchmark/mapping.yml

```diff
@@ -3,49 +3,58 @@ Accounts:
     table: accounts
     fields:
         Name: name
-    record_type: Organization
```

Comment:
Is Organization the default Account record type?

Reply:
I actually don't know, but I'm not sure if it matters. What happens if they default to the wrong record type?

I'd expect NPSP to "prefer" Household anyhow.

Reply:
NPSP (at least in the most common configuration) will automatically create a Household account for contacts created without an explicit account relation. Accounts can also be created with the Organization record type to represent organizations that are giving money. Since the contacts in this file are created without any lookup to these accounts, I assume these accounts are meant to be the latter. I doubt it makes a difference for performance but I'm pointing out a place where the data may not be realistic, because I'm not sure what the default record type will be.

Reply:
Would you suggest going back to record_type: Organization or Household?

Reply:
Organization. You'll have Households generated automatically for the contacts that are inserted, like I said.

Reply:
https://github.com/SalesforceFoundation/Cumulus/pull/4592/commits/fd9b6892b9f7da5954b00a074a72af3d9c677dca

I'm mostly only interested in realism from a performance perspective (I don't even use Faker for names) but its an easy fix.